### PR TITLE
Update syncSLErepos

### PR DESCRIPTION
**Pull request by toabctl**

### CODE REVIEW

File: hostscripts/clouddata/syncSLErepos [hunk 1]
```diff
@@ -1,5 +1,23 @@
 #!/bin/bash
 
+# SITE to use. Can be "nbr" or "provo"
+: ${SITE:="nbr"}
+
+if [ $SITE = "nbr" ]; then
```
Comment (dirkmueller) [inline hostscripts/clouddata/syncSLErepos]: I personally prefer nue and prv (which matches the $site.suse.net domains)

File: hostscripts/clouddata/syncSLErepos [hunk 2]
```diff
@@ -1,5 +1,22 @@
 #!/bin/bash
-r=/srv/nfs/repos/
+
+# SITE to use. Can be "nbr" or "provo"
+: ${SITE:="nbr"}
```
Comment (dirkmueller) [inline hostscripts/clouddata/syncSLErepos]: nue and prv

File: hostscripts/clouddata/syncSLErepos (hunk 1 above)
Comment (toabctl) [inline hostscripts/clouddata/syncSLErepos]: Done

File: hostscripts/clouddata/syncSLErepos (hunk 2 above)
Comment (toabctl) [inline hostscripts/clouddata/syncSLErepos]: Done

File: hostscripts/clouddata/syncSLErepos [hunk 3]
```diff
@@ -44,56 +59,74 @@ for servicepack in 1 2 3; do
...
             if [ -z "$sync_from_ibs" ]; then
-                $rsync $buildsuse/SUSE/Products/SLE-HA/$version/$arch/product/ /srv/nfs/repos/$arch/SLE$repo_version-HA-Pool/
+                $rsync ${ibs_source}/SUSE/Products/SLE-HA/$version/$arch/product/ $destdir_ha_pool
             else
-                $rsync /mnt/dist/ibs/SUSE\:/SLE-$version\:/GA/images/repo/SLE-$version-HA-POOL-$arch-Media1/ /srv/nfs/repos/$arch/SLE$repo_version-HA-Pool/
+                $rsync /mnt/dist/ibs/SUSE\:/SLE-$version\:/GA/images/repo/SLE-$version-HA-POOL-$arch-Media1/ $destdir_ha_pool
             fi
             # HA updates
-            $rsync $buildsuse/SUSE/Updates/SLE-HA/$version/$arch/update/ /srv/nfs/repos/$arch/SLE$repo_version-HA-Updates/
+            desdir_ha_updates=/srv/nfs/repos/$arch/SLE$repo_version-HA-Updates/
```
Comment (dirkmueller) [inline hostscripts/clouddata/syncSLErepos]: destdir?

File: hostscripts/clouddata/syncSLErepos [hunk 4]
```diff
@@ -44,56 +59,74 @@ for servicepack in 1 2 3; do
...
+                desdir_ha_updates_test=/srv/nfs/repos/$arch/SLE$repo_version-HA-Updates-test/
+                mkdir -p $desdir_ha_updates_test
+                $rsync $ibsmaint/SLE-HA\:/$version\:/$arch/update/ $desdir_ha_updates_test
             fi
         fi
 
         # SDK is only for > SP1, and not for ARM
         if [ $servicepack -gt 0 -a "$arch" != aarch64 ]; then
             # SDK pool
+            desdir_sdk_pool=/srv/nfs/repos/$arch/SLE$repo_version-SDK-Pool/
```
Comment (dirkmueller) [inline hostscripts/clouddata/syncSLErepos]: destdir?

Comment (toabctl) [inline hostscripts/clouddata/syncSLErepos]: done

File: hostscripts/clouddata/syncSLErepos (hunk 3 above)
Comment (toabctl) [inline hostscripts/clouddata/syncSLErepos]: done

**Comment by aspiers** [comment]
Updated the PR title to avoid confusion with `sync-repos` (which was deleted earlier today).  Would also be really nice if this PR (and all others) had a description, *especially* without having to ask for it ;-)

File: hostscripts/clouddata/syncSLErepos [hunk 5]
```diff
@@ -1,5 +1,23 @@
...
+    # Currently /mnt/dist can be setup via: "mount dist.nue.suse.com:/dist /mnt/dist/"
+    ibs_source="/mnt/dist/ibs/"
+    # FIXME(toabctl): use rsync for ibs_source and install_source when
+    # https://infra.nue.suse.com/SelfService/Display.html?id=90326 is fixed
+    #ibs_source="rsync://dist.suse.de/ibs/"
+    #install_source="/mnt/dist/install/"
+elif [ $SITE = "prv" ]; then
+    ibs_source="rsync://ibs-mirror.prv.suse.net/dist/ibs/"
+    #install_source="rsync://ibs-mirror.prv.suse.net/dist/install/"
+else
```
Comment (bmwiedemann) [inline hostscripts/clouddata/syncSLErepos]: could be written as a `case $SITE`

**Pull request merged**